src/app/utils/plug-descriptions.ts

```diff
@@ -85,29 +85,9 @@ export function usePlugDescriptions(
   const perks = getPerkDescriptions(plug, defs, statAndBungieDescStrings);
 
   if (showCommunityDescription && allClarityDescriptions) {
-    let clarityPerk = allClarityDescriptions[plug.hash];
-
-    // if we couldn't find a Clarity description for this perk, fall back to the non-enhanced perk variant
-    if (!clarityPerk) {
-      const regularPerkHash = enhancedPerkToRegularPerk[plug.hash];
-      if (regularPerkHash) {
-        clarityPerk = allClarityDescriptions[regularPerkHash];
-      }
-    }
```

Comment:
Can we reinstate this logic? This lets DIM fall back to displaying the community description for the regular perk if there isn't a community description for the enhanced perk, as is currently the case for **Perpetual Motion**, **Moving Target** etc.:

![image](https://user-images.githubusercontent.com/17512262/201503736-cac10c47-c561-42dd-bcee-499e2527d803.png)

Reply:
I would prefer to see the problem and fix it instead of hiding it
Especially then fix is just a few clicks

Reply:
Sure, this would mask the upstream issue, but I would argue that the upstream issue should be caught closer to the source (i.e. the process that generates the descriptions) rather than relying on the consumer (DIM) to highlight issues. Presumably, a unit test on Clarity's side that highlights missing descriptions for enhanced perks would be a faster, less error-prone method of identifying these issues vs. manually inspecting every perk's tooltip?

Reply:
The only reason I have not automated this is that I'm not sure how I can be sure how to link enhanced with normal perks
I can't just use names because for example where are 3 rampage perks normal, exotic, and enhanced

Reply:
Could possibly source the lookup from [here](https://github.com/DestinyItemManager/d2-additional-info/blob/master/output/trait-to-enhanced-trait.json) (which is automatically regenerated when the definitions are updated and is what DIM uses) or alternatively, use the [code](https://github.com/DestinyItemManager/d2-additional-info/blob/master/src/generate-trait-enhanced-trait-lookup.ts) directly.